rackspace/metrics/v2/ingestion/requests.go

```diff
@@ -0,0 +1,205 @@
+package ingestion
+
+import (
+	"github.com/rackspace/gophercloud"
+	"fmt"
+)
+
+// SendMetrics will ingest the metrics for the tenant associated with RackspaceMetrics.
+func SendMetrics(c *gophercloud.ServiceClient, metrics []MetricData) {
+	var res PostResult
+	reqBody := make([]map[string]interface{}, len(metrics))
+	for i := range metrics {
+		Metric := metrics[i]
+		reqBody[i] = map[string]interface{}{
+			"collectionTime": Metric.CollectionTime,
+			"ttlInSeconds":Metric.TtlInSeconds,
+			"metricValue": Metric.MetricValue,
+			"metricName": Metric.MetricName,
```

Comment:
Whether or not map[string]interface{} is the best solution or not will depend on how that construct is actually used.  Gophercloud has changed significantly since I last maintained it, but since I was asked for help, I'd like to be brought up to speed on how you intend on using this map.  Why is it being produced?  What's consuming it?  Etc.